src/dfx/src/commands/new.rs

```diff
@@ -63,6 +66,77 @@ pub struct NewOpts {
     /// NPM to decide.
     #[arg(long, requires("frontend"))]
     agent_version: Option<String>,
+
+    #[arg(long, value_enum)]
+    extras: Vec<Extra>,
+}
+
+#[derive(ValueEnum, Debug, Copy, Clone, PartialEq, Eq)]
+enum BackendType {
+    Motoko,
+    Rust,
+    Azle,
+    Kybra,
+}
+
+impl Display for BackendType {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Motoko => "Motoko",
+            Self::Rust => "Rust",
+            Self::Azle => "TypeScript (Azle)",
+            Self::Kybra => "Python (Kybra)",
+        }
+        .fmt(f)
+    }
+}
+
+#[derive(ValueEnum, Debug, Copy, Clone, PartialEq, Eq)]
+enum FrontendType {
+    Svelte,
```

Comment:
to be more accurate, can we use sveltekit instead of svelte here?

Reply:
@krpeacock do you have an opinion on this?

Reply:
@adamspofford-dfinity In the absence of an opinion from anyone with a better handle on the ecosystem, I think we should change this to SvelteKit (and the cli option to `--sveltekit`), since there is a distinction between the two.